internal/core/data/v2/controller/http/event_test.go

```diff
@@ -289,3 +289,32 @@ func TestEventById(t *testing.T) {
 		})
 	}
 }
+
+func TestEventTotalCount(t *testing.T) {
+	dbClientMock := &dbMock.DBClient{}
+	dbClientMock.On("EventTotalCount").Return(uint32(0), nil)
```

Comment:
Use a none zero expected count, since 0 is the default value for an uninitialized uint32

Reply:
modified